### PR TITLE
Add "allowed syntax" section to documentation

### PR DESCRIPTION
**Pull request by richardhj**
I started documenting the allowed yaml syntax.

When parsed, the table will look like:

Screen Shot 2019-10-04 at 10 30 06

### CODE REVIEW

**Comment by aschempp** [comment]
Nice! Should we have a column with txpe info as well?

File: docs/de/README.md [hunk 1]
```diff
@@ -43,6 +43,17 @@ de:
...
+## YAML-Syntax
+
+In der YAML-Datei können die folgenden Keywörter definiert werden: 
+
+| | | 
+|-|-| 
+| __title__       | Der Titel der Erweiterung | 
+| __description__ | Lange Beschreibung der Extension, die in der Detailansicht erscheint | 
+| __keywords__    | Liste von Keywörtern für die Suche | 
+| __dependency__  | Bei `true` wird die Extension nicht über die Suche gefunden. Dies ist der Fall für Extensions, die nicht für den Endnutzer gedacht sind (Standaard: false) |
```
Comment (Toflar) [inline docs/de/README.md]:
```suggestion
| __dependency__  | Bei `true` wird die Extension nicht über die Suche gefunden. Dies ist der Fall für Extensions, die nicht für den Endnutzer gedacht sind (Standard: `false`) | 
```

File: docs/en/README.md [hunk 2]
```diff
@@ -44,6 +44,17 @@ en:
...
+## YAML syntax
+
+The following keywords can be defined in the meta data yaml file:
+
+| | | 
+|-|-| 
+| __title__       | Title of the extension | 
+| __description__ | Long description of the extension that shows up in the Details view | 
+| __keywords__    | List of keywords that will enhance search performance  | 
+| __dependency__  | If true, the extension will not show up in the search. This is useful for extensions not tailored for end-users (default: false) |
```
Comment (Toflar) [inline docs/en/README.md]:
```suggestion
| __dependency__  | If true, the extension will not show up in the search. This is useful for extensions not tailored for end-users (default: `false`) | 
```

**Comment by richardhj** [comment]
> Nice! Should we have a column with txpe info as well?

Don't think so, as it is self-explanatory in all cases

File: docs/de/README.md [hunk 3]
```diff
@@ -43,6 +43,17 @@ de:
 Bitte verwendet 4 Leerzeichen zum Einrücken oder Verschachteln und sprecht den Nutzer mit "du" an.
 Für weitere Informationen zum Dateiformat, sieh dir die [Transifex Dokumentation][2] an.
 
+## YAML-Syntax
+
+In der YAML-Datei können die folgenden Keywörter definiert werden: 
+
+| | | 
+|-|-| 
+| __title__       | Der Titel der Erweiterung |
```
Comment (xchs) [inline docs/de/README.md]:
```suggestion
| __title__       | Der Titel der Extension| 
```

**Comment by aschempp** [comment]
Ist `support` from Linter bereits unterstützt?

**Pull request merged**